sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1309,9 +1301,13 @@ private boolean shouldRefreshDialog(@Nullable Dialog dialog) {
     /**
      * Show dialog and remember which activity displayed it for later U.I. state change.
      *
-     * @param dialog dialog.
+     * @param dialog A dialog. It can be null in cases when it wasn't been created
+     *               (for example progress dialog is required only for mandatory updates).
      */
-    private void showAndRememberDialogActivity(Dialog dialog) {
+    private void showAndRememberDialogActivity(@Nullable Dialog dialog) {
+        if (dialog == null) {
```

Comment:
Before it was never called without a dialog, why would it be now?

Reply:
isMandatory check is moved to encapsulate progress dialog logic